agent/installer/installer.go

```diff
@@ -32,6 +32,14 @@ const (
 	ErrBundleUninstall = Error("Error uninstalling bundle")
 )
 
+// BundleType is used to support various bundles
+type bundleType string
+
+const (
+	// BundleTypeK8s represents a vanilla k8s bundle
+	BundleTypeK8s bundleType = "k8s"
```

Comment:
I'm assuming in future we would get this field from cluster-template and we remove this `const`, correct?

Reply:
yup some refactoring will be required at that point in time, but this could be used in case there is no 1:1 mapping.